actions/class.PropertiesAuthoring.php

```diff
@@ -304,6 +304,29 @@ public function getClassForm(core_kernel_classes_Class $class): tao_helpers_form
                 //save all properties values
                 if (isset($data['properties'])) {
                     $this->saveProperties($data);
+                    
+                    $elementRangeArray = [];
+                    $groups = $myForm->getGroups();
+                    if (isset($data['properties'])) {
+                        foreach ($data['properties'] as $prop) {
+                            if ($prop['range'] != "") {
+                                $index = strstr($groups['property_' . $prop["uri"]]['elements'][0], '_', true);
+                                $elementRangeArray[$index . "_range_list"] = $prop["range"];
```

Comment:
1. Don't use double quotes if you don't need them
2. Are you sure, that `$prop['uri']['elements'][0]` exists every time?

```suggestion
                            if (!empty($prop['range'])) {
                                $index = strstr($groups['property_' . $prop['uri']]['elements'][0], '_', true);
                                $elementRangeArray[$index . '_range_list'] = $prop['range'];
```

Reply:
Done. It seems that it exists every time. :-)